### DISCUSSION TITLE
Pixelated rendering results on android devices

### DISCUSSION THREAD

**Discussion by hannojg** [google/filament#7729] [Q&A]
Hey, before filling out an issue template I wanted to see if anyone can help me with this issue.

We are using filament primarily for 3D rendering on android, and it's working pretty well so far!
However, on some android devices (Pixel 7 Pro, Pixel 7a) some users have reported issue with the rendering output appearing pixelated.

Here is a screenshot of such cases:

<img width="291" alt="Screenshot 2024-04-02 at 10 26 58" src="https://github.com/google/filament/assets/16821682/4b38d355-f8a6-491e-889a-57d326000edb">

<img width="258" alt="Screenshot 2024-04-02 at 10 26 43" src="https://github.com/google/filament/assets/16821682/aa99f1e5-20f8-4ea6-8ab0-e8f707d39b48">

We are using the clear option to render the filament surface with opacity.

has anyone of you experienced something like this and knows what area of the rendering process could affect this?

(I ofc know that a full reproduction would be the best, I am working on that!)

**Comment by romainguy** [comment]
Have you tried enabling/disabling various post process features to see if any of them is responsible for the problem? (bloom, fxaa, etc.)

**Reply by hannojg** [reply]
No, are they set / enabled by default? If so, I will give it a try to disable them!

**Reply by hannojg** [reply]
Okay so I disabled all post processing effects. That changes nothing.
However when I did `view->setPostProcessingEnabled(false)` it rendered correctly.
So it must be some post processing effect that I can't control via the view's API that causes this issue/

Also it works when using Vulkan as backend (in this case I can keep the post processing effects enabled).

**Reply by hannojg** [reply]
What's interesting is that I got the same error on some none pixel devices if I use the Vulkan backend instead of openGL.

**Reply by hannojg** [reply]
I am able to reproduce these issues in some of the samples - will open a new issue ticket!

**Reply by hannojg** [reply]
https://github.com/google/filament/issues/7794
